plugins/auth-backend-module-oidc-proxy-provider/src/index.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export { authModuleOidcProxyProvider as default } from './module';
+export { oidcProxySignInResolvers } from './resolvers';
```

Comment:
you typically want to export the underlying authenticator too, that's [what others do](https://github.com/backstage/backstage/blob/732583700dd0809dae4614792bd67177d7c9246e/plugins/auth-backend-module-oauth2-proxy-provider/src/index.ts#L24) so that people can build their own implementations of your provider

Reply:
Exported to align with the others.